Source/WebCore/html/HTMLScriptElement.h

```diff
@@ -85,6 +88,9 @@ class HTMLScriptElement final : public HTMLElement, public ScriptElement {
     bool isScriptPreventedByAttributes() const final;
 
     Ref<Element> cloneElementWithoutAttributesAndChildren(Document&) final;
+
+    // https://w3c.github.io/trusted-types/dist/spec/#dom-htmlscriptelement-scripttext-slot
+    String m_scriptText;
```

Comment:
Why is this needed?

Reply:
A follow up PR will make use of this slot for some trusted types enforcement. Happy to remove the slot from this PR though.

Reply:
Please remove that from this PR.